### DISCUSSION TITLE
DELTA 2

### DISCUSSION THREAD

**Discussion by mattenausohz** [vwt12eh8/hassio-ecoflow#56] [General]
Hello there!

Are there any plans yet to support DELTA 2? Just received one and it looks like it is not recognized yet.

**Comment by emmvaueff** [comment]
I also have a Delta 2 with additional battery device and need to use this cool integration.

**Comment by Gixgi** [comment]
Hi there
Thanks for the great work, Delta 2 support would be appreciated very much.

**Comment by harambe88** [comment]
Another Delta 2 user here!

**Comment by hapkey** [comment]
and another Delta 2 user is hoping for an integration

**Comment by bastianveiser** [comment]
Delta 2 support would be fantastic!

**Comment by rainepretorius** [comment]
Delta 2 would be amazing

**Comment by DerBaer1986** [comment]
Delta 2 Max here
